### ISSUE TITLE
Engineering borgs cant open up biogens or hydroponic 

### ISSUE THREAD

**Issue by misterreporter** [Skyrat-SS13/Skyrat-tg#2151]
## Reproduction:
1.  Load up the engineering module as a cyborg.

2. Try to use your screwdriver on a biogen or a hydoponics tray. You will hit it instead of opening their maintance panels to upgrade them or disassemble.

3. This doesnt appear to happen to any other type of machinery on the station.

**Comment by davidchan13** [comment]
I believe you need to remove the biogen's beaker before you can open the panel.
As for the trays, not sure why those are being stingy, did you try to wrench them first to see if that would you?

**Comment by misterreporter** [comment]
> I believe you need to remove the biogen's beaker before you can open the panel.
> As for the trays, not sure why those are being stingy, did you try to wrench them first to see if that would you?

For the biogen the beaker was removed, and my screwdriver still failed to work.

For the trays I did not try wrenching, however, you do not need to wrench a hydro tray to upgrade it as an organic mob either. A crew member was able to screw driver them open as it should of been intended, same thing for the biogen, this problem seems to resolve around the engineering cyborgs screw driver for some reason.
